GUI/Main/MainRepo.cs

```diff
@@ -97,6 +97,11 @@ private void UpdateRepo(object sender, DoWorkEventArgs e)
             {
                 errorDialog.ShowErrorDialog(ex.ToString());
             }
+            catch (ReinstallModuleKraken)
+            {
+                // Bypass the generic error dialog so the Post can handle this
+                throw;
```

Comment:
I thought to remember that we did this (catch-rethrow) somewhere else previously and it caused compiler warnings.
But I don't see any here, so I guess we're fine?

Reply:
Maybe in that other case we were re-throwing _all_ exceptions, which would be equivalent to removing the `catch` statements completely?

Reply:
Ah good point, maybe the compiler is smart enough to detect that not handling it would change the code path due to the general `catch (Exception)` below, and this catch-rethrow prevents it for this exception :bulb: